pkg/lib/operation/local/create/config/operation.go

```diff
@@ -0,0 +1,84 @@
+package config
```

Comment:
1. commit, definicia operacii:

Z `CLI` balicka vytiahnuty kod + doplnene `options` a `dependencies`.
Bez interakcie s uzivatelom a bez ineho kodu, ktory sem neparti.


![image](https://user-images.githubusercontent.com/19371734/143893719-4e5eba64-ebc6-430d-b93e-90b0859aba7d.png)

Reply:
Nie su tu ziadne nove testy, .... je to pokryte funkcionalnymi testami, ... prakticky by sa tu museli vsetky zbytocne skopirovat. S tym rozdielom, ze vstup vy nebol `CLI` prikaz, ale struktura `Options`.